qubesadmin/tools/qvm_template.py

```diff
@@ -573,9 +576,14 @@ def qrexec_repoquery(
                 raise ValueError
             dlsize = int(dlsize)
             # First verify that the date does not look weird, then parse it
-            if not re.fullmatch(date_re, buildtime):
+            if re.fullmatch(date_re, buildtime):
+                buildtime = datetime.datetime.strptime(buildtime, \
+                        '%Y-%m-%d %H:%M')
+            elif buildtime.isnumeric():
+                # DNF5 provides seconds since epoch
+                buildtime = datetime.datetime.fromtimestamp(int(buildtime))
```

Comment:
Going further - can DNF5 be convinced to the old timestamp format?

Reply:
> Going further - can DNF5 be convinced to the old timestamp format?

This appears to be impossible. There is only `%{buildtime}`. Help says: `Display buildtime of the package in Unix time`.

Reply:
I see... I don't like that cross-VM data format is changing here, but since the new one is easier to handle, I'm going to accept it.